### ISSUE TITLE
OpenSSL segfault on CentOS 7

### ISSUE THREAD

**Issue by HelloZeroNet** [HelloZeroNet/ZeroNet#132]
Probably because of the missing ECDA support.
```
# python opensslVerify.py
fish: Job 1, 'python opensslVerify.py' terminated by signal SIGSEGV (Address boundary error)
```
It's not possible to catch SIGSEGV, so it would be nice to detect the missing support and fall back to pure-python solution.

**Comment by HelloZeroNet** [comment]
possible solution:
```
cd /usr/local/src
wget -qO- http://www.openssl.org/source/openssl-1.0.1f.tar.gz | tar xzv
cd openssl-1.0.1f
./config shared --prefix=/usr/local --openssldir=/usr/local/ssl
make && make install
```

**Comment by HelloZeroNet** [comment]
Fixed in: 0ca0c754e677b0d2a2dad939d678875a24c15c89
